### PR TITLE
add 2020-09-09-abrt-new-name.markdown

### PR DESCRIPTION
**Pull request by xsuchy**

### CODE REVIEW

File: _posts/2020-09-09-abrt-new-name.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+layout: post
+title:  "New name for ABRT?"
+date:   2020-09-09 10:00:00 +0100
+author: Miroslav Suchý
+visible: 1
+categories:
+---
+The project ABRT started in 2019. The initial name was `CrashWatcher`. Very quickly changed to `CrashCatcher`. But in one month, it got its final name ABRT. ABRT is [name of POSIX signal](https://en.wikipedia.org/wiki/Signal_(IPC)#POSIX_signals) and stems from the word `abort`.
+
+ABRT project was meant as a tool to easy the life of Red Hat Support. Unfortunately Red Hat Support never fully utilized and used ABRT (with some minor exceptions). I recently analyzed the use of ABRT, and its strength are for developers and DevOps. We can identify and helps to report bugs when new software or major release is released. Devops can leverage that we can identify crashes in their deployments and show it in a private instance of ABRT Analytics.
```
Comment (ernestask) [inline _posts/2020-09-09-abrt-new-name.markdown]: s/easy/ease/

File: _posts/2020-09-09-abrt-new-name.markdown [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title:  "New name for ABRT?"
+date:   2020-09-09 10:00:00 +0100
+author: Miroslav Suchý
+visible: 1
+categories:
+---
+The project ABRT started in 2019. The initial name was `CrashWatcher`. Very quickly changed to `CrashCatcher`. But in one month, it got its final name ABRT. ABRT is [name of POSIX signal](https://en.wikipedia.org/wiki/Signal_(IPC)#POSIX_signals) and stems from the word `abort`.
```
Comment (ernestask) [inline _posts/2020-09-09-abrt-new-name.markdown]: ABRT is *the* name

Comment (ernestask) [reply _posts/2020-09-09-abrt-new-name.markdown]: of *a* POSIX signal

File: _posts/2020-09-09-abrt-new-name.markdown [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+ABRT project was meant as a tool to easy the life of Red Hat Support. Unfortunately Red Hat Support never fully utilized and used ABRT (with some minor exceptions). I recently analyzed the use of ABRT, and its strength are for developers and DevOps. We can identify and helps to report bugs when new software or major release is released. Devops can leverage that we can identify crashes in their deployments and show it in a private instance of ABRT Analytics.
+
+So we have a clear shift in the use.
+
+The name of ABRT has several cons: It is hard to pronounce. And a lot of people still remember when ABRT was unable to deduplicate bugs and spammed Bugzilla in its early days.
```
Comment (ernestask) [inline _posts/2020-09-09-abrt-new-name.markdown]: The “of” looks off here. s/of ABRT/“ABRT”/.

File: _posts/2020-09-09-abrt-new-name.markdown [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+Therefore the ABRT team would like to rename ABRT to ... something else. To indicate new directions: orientation on developers and DevOps. But we struggle what the name should be.
+
+Can you help us? Please [file in this form](https://docs.google.com/forms/d/e/1FAIpQLSdQfpyndofp1oIIQJ58wXuNjDXyyGbFgOuAJIubA_F8aR_83g/viewform).
```
Comment (ernestask) [inline _posts/2020-09-09-abrt-new-name.markdown:19]: s/file/fill/

File: _posts/2020-09-09-abrt-new-name.markdown [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+The name should not be used in different SW project (this outs `CrashCatcher). It should be easy to pronounce. The name may be related to the function - collecting crashes.
```
Comment (ernestask) [inline _posts/2020-09-09-abrt-new-name.markdown]: “by another SW project” or “in some different SW project”. Definitely could at least use an article.

File: _posts/2020-09-09-abrt-new-name.markdown (hunk 4 above)
Comment (xsuchy) [reply _posts/2020-09-09-abrt-new-name.markdown:19]: I believe that this one is correct.

**Comment by xsuchy** [comment]
updated

File: _posts/2020-09-09-abrt-new-name.markdown (hunk 4 above)
Comment (mzidek-gh) [reply _posts/2020-09-09-abrt-new-name.markdown:19]: "Fill in" is the correct phrasal verb ('fill in', 'fill out' and 'fill up' should be all usable here).

File: _posts/2020-09-09-abrt-new-name.markdown (hunk 1 above)
Comment (ari3s) [reply _posts/2020-09-09-abrt-new-name.markdown]: @ernestask I think that the project started in 2009.

Comment (ernestask) [reply _posts/2020-09-09-abrt-new-name.markdown]: Wrong person, but good catch.

Comment (ari3s) [reply _posts/2020-09-09-abrt-new-name.markdown]: @ernestask I am sorry, it should be addresses to @xsuchy.

**Pull request merged**